openwisp_utils/qa.py

```diff
@@ -51,3 +51,147 @@ def check_migration_name():
         raise Exception("Migration %s %s in directory %s must "
                         "be renamed to something more descriptive."
                         % (file_, ', '.join(migrations), args.migration_path))
+
+
+def _commit_check_args():
+    """
+    Parse and return CLI arguments
+    """
+    parser = argparse.ArgumentParser(description="Ensures migration files "
```

Comment:
The description of this argument seems like it's been copied from above! Please check this!

Reply:
```
description="Ensures the commit message "
"matches the OpenWISP commit guidelines.")
```